appveyor.yml

```diff
@@ -0,0 +1,23 @@
+before_build:
+  - nuget restore
+
+test: off
+
+image: Visual Studio 2017
+
+build:
+  project: Agoda.LoadBalancing.sln
+
+after_build:
+  - ps: .\tools\publish.ps1
+
+artifacts:
+  - path: '**\*.nupkg'
+    name: NugetPackage
+  
+deploy:
+  provider: NuGet
+  api_key:
+    secure: g5JNOfRkIpNjLPxGk2yfmG+RLPRkhqT71N11Hfx9ieRb90bUnPl0grIJpmFvwz7k
```

Comment:
Is this secret?

Reply:
yes it the is encrypted value `Your NuGet API key should be encrypted using this tool: https://ci.appveyor.com/tools/encrypt.`